lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -16,28 +16,42 @@ def identifier
     end
 
     def document
+      {}.merge(common_fields)
+        .merge(fields_from_expanded_links)
```

Comment:
@dwhenry Could you give an example? Do you mean creating intermediate objects for different parts of the response, and then serialising those?